libcxx/include/__ranges/movable_box.h

```diff
@@ -134,6 +134,13 @@ concept __doesnt_need_empty_state =
          // 2. Otherwise, movable-box<T> should store only a T if either T models movable or
```

Comment:
Not attached to this line: We need to add a release note in the ABI Changes section. You can add something as you think and I can help with the wording if you're unsure how much detail we want to include.